tests/NormalizerTest.php

```diff
@@ -309,6 +311,35 @@ public function testFormatExceptions() {
 		);
 	}
 
+	public function testFormatExceptionWithPreviousThrowable() {
+		$t = new TypeError("not a type error");
+		$e = new Exception("an exception", 13, $t);
```

Comment:
```suggestion
		$t = new TypeError('not a type error');
		$e = new Exception('an exception', 13, $t);
```

Reply:
I want to add the cs fixer later on :wink: